### PR TITLE
Capi: Shape bounds binding

### PR DESCRIPTION
**Pull request by mmaciola**
Added Shape->bounds Capi binding

### CODE REVIEW

**Review by JSUYA — COMMENTED** [review]
In thorvg.h, `bounds` method is declared in `Paint` class.
I think capi should also follow this structure.
What do you think?

File: inc/thorvg_capi.h [hunk 1]
```diff
@@ -163,6 +163,7 @@ TVG_EXPORT Tvg_Result tvg_shape_get_fill_rule(const Tvg_Paint* paint, Tvg_Fill_R
 TVG_EXPORT Tvg_Result tvg_shape_set_linear_gradient(Tvg_Paint* paint, Tvg_Gradient* grad);
 TVG_EXPORT Tvg_Result tvg_shape_set_radial_gradient(Tvg_Paint* paint, Tvg_Gradient* grad);
 TVG_EXPORT Tvg_Result tvg_shape_get_gradient(const Tvg_Paint* paint, Tvg_Gradient** grad);
+TVG_EXPORT Tvg_Result tvg_shape_get_bounds(Tvg_Paint* paint, float* x, float* y, float* w, float* h);
```
Comment (hermet) [inline inc/thorvg_capi.h:166]: => Tvg_Result tvg_paint_get_bounds(const Tvg_Paint* paint, float* x, float* y, float* w, float* h);

**Review by hermet — CHANGES_REQUESTED** [review]

**Review by JSUYA — APPROVED** [review]
LGTM

File: inc/thorvg_capi.h [hunk 2]
```diff
@@ -130,6 +130,7 @@ TVG_EXPORT Tvg_Result tvg_paint_transform(Tvg_Paint* paint, const Tvg_Matrix* m)
 TVG_EXPORT Tvg_Result tvg_paint_set_opacity(Tvg_Paint* paint, uint8_t opacity);
 TVG_EXPORT Tvg_Result tvg_paint_get_opacity(Tvg_Paint* paint, uint8_t* opacity);
 TVG_EXPORT Tvg_Paint* tvg_paint_duplicate(Tvg_Paint* paint);
+TVG_EXPORT Tvg_Result tvg_paint_get_bounds(Tvg_Paint* paint, float* x, float* y, float* w, float* h);
```
Comment (hermet) [inline inc/thorvg_capi.h:133]: Tvg_Paint* paint => const Tvg_Paint* paint,

**Review by hermet — CHANGES_REQUESTED** [review]
please check a comment.

**Review by hermet — APPROVED** [review]

**Pull request merged**